pr2eus_moveit/euslisp/robot-moveit.l

```diff
@@ -463,18 +463,44 @@
                     cds confkey :end-coords ed-lst args))
      ret))
   (:angle-vector-motion-plan ;;
-   (av &rest args &key (move-arm :larm) (reset-total-time 5000.0) (use-send-angle-vector) &allow-other-keys)
-   (let (traj ret)
+   (av &rest args &key (start-offset-time) (total-time) (move-arm :larm) (reset-total-time 5000.0) (use-send-angle-vector) &allow-other-keys)
+   (let (traj ret filtered orig-total-time)
      (setq ret (send* self :angle-vector-make-trajectory av args))
      (when ret
        (setq traj (send ret :trajectory :joint_trajectory))
-       (when (< (send (send (car (last (send traj :points))) :time_from_start) :to-sec) 0.001)
+       (setq orig-total-time (send (send (car (last (send traj :points))) :time_from_start) :to-sec))
+       (when (< orig-total-time 0.001)
          (unless reset-total-time
            (ros::ros-error "Trajectory has very short duration")
            (return-from :angle-vector-motion-plan nil))
          (ros::ros-warn "reset Trajectory Total time")
```

Comment:
@YoheiKakiuchi Do we need `reset-total-time` feature, if we have https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/252/files#diff-113ce1b47d7cec61152334f1050188c1R479 feature? which always try to scale planned trajectory to use setting 'tm' value?